### ISSUE TITLE
dumpTable with page=0 fails

### ISSUE THREAD

**Issue by kyrias** [hyperboria/python-cjdns#8]
Trying to use `NodeStore_dumpTable()` with `page=0` fails with `{u'txid': u'0UEZDK65EY', u'error': u'Entry [page] is required and must be of type [Int]'}`, but trying to run the same query with `cexec` works.

**Comment by kyrias** [comment]
```
09:38:02.338879 IP 127.0.0.1.48879 > 127.0.0.1.11234: UDP, length 158
    0x0000:  4500 00ba 1468 4000 4011 27c9 7f00 0001  E....h@.@.'.....
    0x0010:  7f00 0001 beef 2be2 00a6 feb9 6432 3a61  ......+.....d2:a
    0x0020:  7131 393a 4e6f 6465 5374 6f72 655f 6475  q19:NodeStore_du
    0x0030:  6d70 5461 626c 6534 3a61 7267 7364 6536  mpTable4:argsde6
    0x0040:  3a63 6f6f 6b69 6531 303a 3134 3033 3433  :cookie10:140343
    0x0050:  3632 3132 343a 6861 7368 3634 3a38 6230  62124:hash64:8b0
    0x0060:  3336 3164 3131 6536 3161 3032 6666 6465  361d11e61a02ffde
    0x0070:  6231 3332 6665 6536 6365 6130 6130 3661  b132fee6cea0a06a
    0x0080:  3331 3037 3364 6365 3161 6133 3764 3264  31073dce1aa37d2d
    0x0090:  3666 6162 3131 6562 3432 3035 3131 3a71  6fab11eb420511:q
    0x00a0:  343a 6175 7468 343a 7478 6964 3130 3a42  4:auth4:txid10:B
    0x00b0:  4644 3059 3257 4248 5765                 FD0Y2WBHWe
```

`{'hash': '8b0361d11e61a02ffdeb132fee6cea0a06a31073dce1aa37d2d6fab11eb42051', 'q': 'auth', 'cookie': '1403436212', 'args': {}, 'txid': 'BFD0Y2WBHW', 'aq': 'NodeStore_dumpTable'}`

So it seems the issue is that the bencode lib skips `page=0`.

**Comment by kyrias** [comment]
Ah, I see:

```
    if isinstance(obj, dict):
        res = "d"
        for key in sorted(obj.keys()):
            if obj[key]:
                res += bencode(key) + bencode(obj[key])
        return res + "e"
```

`0` is false, so it ignores it.

**Comment by kyrias** [comment]
Fixed in d40b334289b2377eff05c003cfb42fc424d49007
